pubnub-chat-api/api/pubnub-chat-api.api

```diff
@@ -402,17 +402,17 @@ public final class com/pubnub/chat/config/PushNotificationsConfig {
 }
 
 public final class com/pubnub/chat/membership/MembersResponse {
-	public fun <init> (Lcom/pubnub/api/models/consumer/objects/PNPage$PNNext;Lcom/pubnub/api/models/consumer/objects/PNPage$PNPrev;IILjava/util/Set;)V
-	public final fun getMembers ()Ljava/util/Set;
+	public fun <init> (Lcom/pubnub/api/models/consumer/objects/PNPage$PNNext;Lcom/pubnub/api/models/consumer/objects/PNPage$PNPrev;IILjava/util/List;)V
```

Comment:
I got an error while running `jvmApiCheck` and got the hint in logs to execute `:pubnub-chat-api:apiDump`.
I ran it and that's why this file has changed. Is it OK?